packages/xchain-ethereum/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Fix
 
 - Fix `estimateGasLimit` to consider memo
+- Fix `getTransactions` self tx duplication issue
```

Comment:
It goes into next version 0.11.2. Please prepare it to bump xchain-eth after merging this PR.

Reply:
0.11.1 wasn't released yet, I forgot it to release last time so I wanted to involve this fix in 0.11.1 and release after that.